docs/src/release_notes/21.8.0.rst

```diff
@@ -0,0 +1,102 @@
+21.8.0
+======
+
+Direct runner on GKE
+--------------------
+
+Klio can now be run on the additional runner of DirectGKERunner.
+
+Under the hood, when using the new DirectGKERunner with a Klio job, Klio creates a GKE deployment. When GKE starts the containers for the deployment, it runs the Klio job in DirectRunner mode (just like you would locally with ``klio job run --direct-runner``.
```

Comment:
We try to write docs using [semantic newlines](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) (although we haven't been good at keeping up with it) - mind adapting?